testing/tests/DevExpress.ui.widgets.pivotGrid/pivotGrid.export.tests.js

```diff
@@ -0,0 +1,182 @@
+        '<xf xfId=\"0\" applyAlignment=\"1\" fontId=\"0\" applyNumberFormat=\"0\" numFmtId=\"0\"><alignment vertical=\"top\" wrapText=\"0\" horizontal=\"right\" /></xf>' +
+        '</cellXfs>' +
+        '<cellStyles count=\"1\"><cellStyle name=\"Normal\" xfId=\"0\" builtinId=\"0\" /></cellStyles>' +
+        '</styleSheet>';
+    const worksheet = '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><worksheet xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="x14ac" xmlns:x14ac="http://schemas.microsoft.com/office/spreadsheetml/2009/9/ac">' +
+        '<sheetPr/><dimension ref="A1:C1"/>' +
+        '<sheetViews><sheetView tabSelected="1" workbookViewId="0"><pane activePane=\"bottomLeft\" state=\"frozen\" xSplit=\"1\" ySplit=\"1\" topLeftCell=\"B2\" /></sheetView></sheetViews>' +
+        '<sheetFormatPr defaultRowHeight="15" outlineLevelRow="0" x14ac:dyDescent="0.25"/>' +
+        '<cols>' +
+        '<col width=\"13.57\" min=\"1\" max=\"1\" />' +
+        '<col width=\"13.57\" min=\"2\" max=\"2\" />' +
+        '<col width=\"13.57\" min=\"3\" max=\"3\" />' +
+        '</cols>' +
+        '<sheetData>' +
+        '<row r=\"1\" spans=\"1:3\" outlineLevel=\"0\" x14ac:dyDescent=\"0.25\"><c r=\"A1\" s=\"0\" t=\"s\" /><c r=\"B1\" s=\"0\" t=\"s\"><v>0</v></c><c r=\"C1\" s=\"0\" t=\"s\"><v>1</v></c></row>' +
+        '<row r=\"2\" spans=\"1:3\" outlineLevel=\"0\" x14ac:dyDescent=\"0.25\"><c r=\"A2\" s=\"1\" t=\"s\"><v>2</v></c><c r=\"B2\" s=\"2\" t=\"s\"><v>3</v></c><c r=\"C2\" s=\"2\" t=\"s\"><v>3</v></c></row>' +
+        '<row r=\"3\" spans=\"1:3\" outlineLevel=\"0\" x14ac:dyDescent=\"0.25\"><c r=\"A3\" s=\"1\" t=\"s\"><v>1</v></c><c r=\"B3\" s=\"2\" t=\"s\"><v>3</v></c><c r=\"C3\" s=\"2\" t=\"s\"><v>3</v></c></row>' +
+        '</sheetData>' +
+        '<ignoredErrors><ignoredError sqref="A1:C3" numberStoredAsText="1" /></ignoredErrors></worksheet>';
+    const sharedStrings = '<?xml version="1.0" encoding="utf-8"?><sst xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main" count="4" uniqueCount="4">' +
+        '<si><t>str2</t></si>' +
+        '<si><t>Grand Total</t></si>' +
+        '<si><t>str1</t></si>' +
+        '<si><t>1</t></si>' +
+        '</sst>';
+
+    testConfiguration(
+        assert,
+        {
+            dataSource: {
+                fields: [{ dataField: 'field1', area: 'row' }, { dataField: 'field2', area: 'column' }, { summaryType: 'count', area: 'data' }],
+                store: [{ field1: 'str1', field2: 'str2' }]
+            }
+        },
+        { styles, worksheet, sharedStrings }
+    );
+});
+
+QUnit.test("Rows: [string, string], Columns: [string, string], Data: sum(number)", function(assert) {
```

Comment:
Add test with several data fields
Add test with dataFieldArea 'row'
Add test with format for data field